e2e/Tests/iothub/service/FileUploadNotificationE2ETest.cs

```diff
@@ -119,48 +119,34 @@ async Task<AcknowledgementType> OnFileUploadNotificationReceived(FileUploadNotif
         }
 
         [TestMethod]
-        [DataRow(IotHubTransportProtocol.Tcp)]
-        [DataRow(IotHubTransportProtocol.WebSocket)]
-        public async Task FileUploadNotification_ErrorProcessor_ReceivesNotifications(IotHubTransportProtocol protocol)
+        public async Task FileUploadNotification_CloseGracefully_DoesNotExecuteConnectionLoss()
         {
-            var options = new IotHubServiceClientOptions
+            // arrange
+            using var sender = new IotHubServiceClient(TestConfiguration.IotHub.ConnectionString);
```

Comment:
Use the cached one since we're not initializing with options. Do not dispose.
```suggestion
            var sender = TestDevice.ServiceClient;
```